com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/Logic/OrNode.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class OrNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,     // Version
+            "Or", // Name
+            "Out = A || B;",
+            new ParameterDescriptor("A", TYPE.Bool, Usage.In),
+            new ParameterDescriptor("B", TYPE.Bool, Usage.In),
```

Comment:
I guess it will be more like a question that do we want TYPE.Bool support vectors.

Reply:
Same answer as above.